soc/arc/snps_nsim/Kconfig

```diff
@@ -9,6 +9,7 @@ choice
 config SOC_NSIM_EM
 	bool "Synopsys ARC EM in nSIM"
 	select CPU_HAS_MPU
+	select CPU_HAS_DSP
 
 config SOC_NSIM_EM7D_V22
 	bool "Synopsys ARC EM7D_V22 in nSIM"
```

Comment:
EM7D is a DSP variant, this one should have the HAS_DSP enabled.

Reply:
Added.